app/styles/nypr-ui/_colors.scss

```diff
@@ -26,3 +26,43 @@ $graywhite: #f8f8f8;            // rgb(248, 248, 248)
 // Social
 $facebook: #3b5998;             // rgb(59, 89, 152)
 $facebook-hover: #2a457c;
+
+// Image Filters
+$filter: rgba(0, 0, 0, .3);
+
+@mixin filter-container {
+    top: 0;
+    left: 0;
+    width: 100%;
+    position: relative;
+}
+
+@mixin classic-filter($filter) {
+    content: "classic filter";
```

Comment:
No particular reason, I saw someone do it and I liked it visually. I'll remove it!